lib/searchkick/index.rb

```diff
@@ -146,34 +135,43 @@ def clean_indices
       indices
     end
 
+    def set_alias_to_index(alias_name, index_name, clear=true)
+      old_indices = begin
+        client.indices.get_alias(name: alias_name).keys
+      rescue Elasticsearch::Transport::Transport::Errors::NotFound
+        []
+      end
+
+      actions = clear ? old_indices.map { |old_name| {remove: {index: old_name, alias: alias_name}} } : []
```

Comment:
I'm pretty sure I tested that situation and it was fine because the add action happens after, but we will double check.